### DISCUSSION TITLE
Welcome to LifeUp Discussions!

### DISCUSSION THREAD

**Discussion by Ayagikei** [Ayagikei/LifeUp#74] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Dai-GoGo** [comment]
2017年我刚上大学，那会儿我刚接触到gamification life的个人管理理念，也想着开发一款和“人升”一样酷的软件，后面一搜应用商店没想到已经有人升了哈哈，一路看着人升到现在的。博主加油💪
